api-spec/api-spec.md

```diff
@@ -1,43 +1,45 @@
 
 The Web Feature Service is a standard API that represents collections of geospatial data. The [Web Feature Service 3.0 API](https://github.com/opengeospatial/WFS_FES), currently under development, is the latest iteration of that standard. WFS 3 defines the RESTful interface to query geospatial data, with GeoJSON as a main return type. With WFS you can return any `Feature`, which is a geometry plus any number of properties. In the STAC specification an `Item` is a `Feature`, with additional required fields for `datetime` and `assets`. WFS also defines the concept of a Collection, which contains `Feature`s. A STAC `Collection` aligns with (and extends slightly) a WFS 3 `Collection`; it contains `Item`s.
 
-In WFS 3 Collections are the sets of data that can be queried ([7.11](https://rawgit.com/opengeospatial/WFS_FES/master/docs/17-069.html#_feature_collections_metadata)), and each describes basic information about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all the data contained. [STAC collections](../collections-spec/README.md) contain this same information, along with other STAC specific fields and thus are compliant with both WFS Collections and STAC Collections and is returned from the `/collections/{collection_id}` endpoint.
+In WFS 3 Collections are the sets of data that can be queried ([7.11](http://docs.opengeospatial.org/DRAFTS/17-069r1.html#_feature_collections)), and each describes basic information about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all the data contained. [STAC collections](../collection-spec/README.md) contain this same information, along with other STAC specific fields and thus are compliant with both WFS Collections and STAC Collections and is returned from the `/collections/{collection_id}` endpoint.
+
+In WFS 3 Features are the individual records within a Collection and are provided in GeoJSON format. [STAC Items](../item-spec/README.md) are analogous to WFS 3 Features, are in GeoJSON, and are returned from the `/collections/{collection_id}/items/{item_id}` endpoint.
+
+## HTTP Request Methods and Content Types
 
-In WFS 3 Features are the individual records within a Collection and are provided in GeoJSON format. [STAC Items](../item-spec/README.md) are analagous to WFS 3 Features, are in GeoJSON, and are returned from the `/collections/{collection_id}/items/{item_id}` endpoint.
+For WFS3-compliance, it is **required** only that `GET` is implemented for the WFS3 endpoints.  Also, it is **required** only that `GET` is implemented for the STAC endpoints. 
 
-### WFS3 Endpoints
+Since STAC adds additional filter parameters that may have much larger values, like `intersects`, it is **recommended** to also support `POST` for both the WFS3 and STAC endpoints that
+ accept filter parameters (e.g., `/collections/{collection_id}/items` and `/stac/search`). 
```

Comment:
I think here we do not want to recommend supporting POST for the collections endpoints as this conflicts with the transactions extension: https://github.com/radiantearth/stac-spec/tree/master/api-spec/extensions/transaction
I believe we want to recommend that for more complex searches that require POST to exclusively use the `/stac/search` endpoint.

Reply:
Agreed.

Reply:
I agree with that as to not conflict with the Transactions Ext as it is currently defined.  

However, I think we should reconsider whether or not Transactions should extend the behavior of the paths defined in WFS 3.  For example, what if an existing WFS 3 implementation has already implemented the GET and POST form-data operations on `/collections/{collectionID}/items`?  I feel that if we want to provide extensions/behaviors on top of the ones defined by WFS 3 that we should duplicate the endpoints, such that, for example, we have  `/collections/{collectionID}/items` that implements only the WFS 3 semantics, and `/stac/collections/{collectionID}/items` that implements our own semantics (e.g., the transactions POST behavior)

Reply:
So originally, before it was an actual extension, I used PUT, PATCH, and DELETE to handle creating, updating, and deleting items against the dynamic api endpoint.  I think it was during the second sprint it was decided that transactions should be executed via the collections endpoint.  I'm not 100% certain why.  @m-mohr is there a WFS-T3 spec that defines transactions against the collections endpoint?  We also have our own catalog endpoints.  Would that make more sense?

Reply:
I don't think there's as much conflict as I originally thought.  I think we just needed to be explicit about the use of the `application/json` .  See the changes in https://github.com/radiantearth/stac-spec/pull/489/commits/e7dbfaaf486e783b949f1226dc42e9edf7a8e764

Reply:
@joshfix Not yet, as far as I know, but there will probably be a WFS-T extension in the future that we could adopt.

Fine with the changes by @philvarner. Approved (again).